package.json

```diff
@@ -50,25 +50,24 @@
         "git-clang-format": "git-clang-format --extensions=ts"
     },
     "dependencies": {
-        "@clangd/install": "0.1.17",
-        "abort-controller": "^3.0.0",
-        "vscode-languageclient": "8.0.2"
+        "@clangd/install": "0.1.20",
+        "vscode-languageclient": "^9.0.1"
     },
     "devDependencies": {
-        "@types/glob": "^7.1.1",
-        "@types/mocha": "^7.0.2",
-        "@types/node": "^6.0.40",
-        "@types/sinon": "^10.0.16",
-        "@types/vscode": "1.65.0",
-        "clang-format": "^1.7.0",
-        "esbuild": "^0.14.13",
-        "glob": "^7.1.4",
-        "mocha": "^9.2.0",
-        "ovsx": "^0.3.0",
-        "sinon": "^15.2.0",
-        "typescript": "^4.5.5",
-        "vsce": "^2.7.0",
-        "vscode-test": "^1.3.0"
+        "@types/glob": "^8.1.0",
+        "@types/mocha": "^10.0.7",
+        "@types/node": "^22.0.2",
+        "@types/sinon": "^17.0.3",
+        "@types/vscode": "1.97.0",
```

Comment:
This seems wrong. I think this needs to match `engines.vscode`, because when I try to run `npm run package` locally, I get:

>  ERROR  @types/vscode 1.97.0 greater than engines.vscode ^1.75.0. Either upgrade engines.vscode or use an older @types/vscode version

Reply:
Matched the engine version.